### PR TITLE
chore: bump react native web to 0.9.0

### PR DESCRIPTION
**Pull request by craigbilner**
`haul` needs `react` `16.5.2` which means we need to go to `react-native-web` `0.9.0` and latest `enzyme`. They needed to be upgraded anyway....

### highlights

* don't need `babel-plugin-react-native-web` in every package
* colours have moved to rgba with a as a float
* some of the other styles are tweaked and less important
* the `moduleNameMapper` key changed slightly
* the react-native-web plugin isn't needed for SSR - rnw has moved to esm and now need to force into cjs. Will address this with the bundling stories as well where we can probably get smaller bundles by using esm throughout for client bundle and cjs for SSR (and IE11)
* enzyme doesn't render `React.Fragment` anymore

tl;dr 

chore: update snaps
chore: remove prop
chore: fix commonjs

### CODE REVIEW

File: packages/article-flag/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prettier:diff": "prettier --list-different '**/*.*'",
     "fmt": "eslint . --fix && prettier --write '**/*.*'",
-    "depcheck": "depcheck --ignores='@thetimes/jest-lint,babel-*,depcheck,eslint,jest,prettier,webpack*' --ignore-bin-package=false --skip-missing",
+    "depcheck": "depcheck --ignores='@thetimes/jest-lint,babel-*,depcheck,eslint,jest,prettier,react-art,webpack*' --ignore-bin-package=false --skip-missing",
```
Comment (colincclark) [inline packages/article-flag/package.json]: react-art does not seem to be in the deps?

File: packages/ssr/webpack.config.dev.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const path = require("path");
 const webpack = require("webpack");
 
-const alias = { "react-native": "react-native-web" };
+const alias = { "react-native$": "react-native-web" };
```
Comment (colincclark) [inline packages/ssr/webpack.config.dev.js]: what does this do? and what is the difference between `react-native$` and `^react-native$`?

File: packages/article-flag/package.json (hunk 1 above)
Comment (craigbilner) [reply packages/article-flag/package.json]: RNW should have been a dev dep, added, ta

File: packages/article-summary/__tests__/web/__snapshots__/article-summary-with-style.web.test.js.snap [hunk 3]
```diff
@@ -18,23 +18,27 @@ exports[`1. article summary component with a single paragraph 1`] = `
     "color": "inherit",
     "font-family": "inherit",
     "font-size": "inherit",
+    "font-weight": "inherit",
+    "line-height": "inherit",
```
[the remark this answers is not in the document]
Comment (craigbilner) [reply packages/article-summary/__tests__/web/__snapshots__/article-summary-with-style.web.test.js.snap]: Raised: https://github.com/newsuk/times-components/issues/1367

File: packages/ssr/webpack.config.dev.js (hunk 2 above)
Comment (craigbilner) [reply packages/ssr/webpack.config.dev.js]: best reference I could find was [this](https://github.com/necolas/react-native-web/issues/961#issuecomment-391164614)

It's just a regex and I'm guessing the `moduleNameMapper` and `alias` are slightly different without looking into it
